@here/harp-mapview/lib/geometry/TileGeometryCreator.ts

```diff
@@ -766,7 +766,14 @@ export class TileGeometryCreator {
                     bufferGeometry.setIndex(attachment.getBufferAttribute(index));
                 }
 
-                if (!bufferGeometry.getAttribute("normal") && needsVertexNormals(technique)) {
+                const isBuilding =
```

Comment:
This may not be immediately clear to other people reading this why a defined edgeIndex means that the normals should be computed, please add a comment here.

Reply:
Adding comment

Reply:
Added comment